include/uv-darwin.h

```diff
@@ -52,6 +57,7 @@
   void* cf_member[2];                                                         \
   int cf_error;                                                               \
   uv_mutex_t cf_mutex;                                                        \
+  enum uv_fs_event_type_flag type_flag;                                       \
```

Comment:
understood. I'll keep playing with it, but I'm not sure if we will be able to keep existing functionality the exact same without knowing if it is a file or directory, so this may have to wait (if at all, of course) until v2.x